schemas/stsci.edu/asdf/transform/constant-1.4.0.yaml

```diff
@@ -0,0 +1,23 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/constant-1.4.0"
+tag: "tag:stsci.edu:asdf/transform/constant-1.4.0"
+title: >
+  A transform that takes no inputs and always outputs a constant
+  value.
+description: |
+  Invertibility: All ASDF tools are required to be able to compute the
+  analytic inverse of this transform, which always outputs zero values.
```

Comment:
Sorry, I don't understand this description. In general Const1(2)D models don't have an analytical inverse.I would suggest
"A transform which takes one or two inputs based on dimensionality and returns a constant value. It has no analytical inverse.